packages/paste-core/components/display-heading/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {Theme} from '@twilio-paste/theme';
+
+import {DisplayHeading} from '../src';
+
+describe('DisplayHeading', () => {
+  it('should render as an h1 when passed as="h1', () => {
+    render(
+      <DisplayHeading as="h1" variant="displayHeading10">
+        test
+      </DisplayHeading>
+    );
+    const renderedDisplayHeading = screen.getByRole('heading', {level: 1});
+    expect(renderedDisplayHeading).toBeDefined();
+  });
+  it('should render as an h2 when passed as="h2', () => {
+    render(
+      <DisplayHeading as="h2" variant="displayHeading20">
+        test
+      </DisplayHeading>
+    );
+    const renderedDisplayHeading = screen.getByRole('heading', {level: 2});
+    expect(renderedDisplayHeading).toBeDefined();
+  });
+  it('should render as an h3 when passed as="h3', () => {
+    render(
+      <DisplayHeading as="h3" variant="displayHeading30">
+        test
+      </DisplayHeading>
+    );
+    const renderedDisplayHeading = screen.getByRole('heading', {level: 3});
+    expect(renderedDisplayHeading).toBeDefined();
+  });
+  it('should remove bottom margin when passed margin="space0', () => {
+    render(
+      <Theme.Provider theme="default">
+        <DisplayHeading as="h3" variant="displayHeading30" marginBottom="space0">
+          test
+        </DisplayHeading>
+      </Theme.Provider>
+    );
+    const renderedDisplayHeading = screen.getByRole('heading', {level: 3});
+    expect(renderedDisplayHeading).toHaveStyle('margin-bottom: 0');
```

Comment:
Issue(Blocking): don't use toHaveStyle, you need to use `toHaveStyleRule('margin-bottom, '0px')`

Reply:
Same with all the customization tests below. toHaveStyle doesn't actually work. If you change the value to something you know is wrong, it will still pass.